ptr_events.py

```diff
@@ -531,6 +534,44 @@ def display_events(self):   # Routine above needs to be called first.
         #     pass
         #eve_skyFlatBegin = sunset +  -32.5/1440  #  NB NB this should come from sro.jscon
         #  NB NB Should add sit time to this report.
+
+
+        eve_skyFlatBegin = sunset +  self.config['eve_sky_flat_sunset_offset']/1440  #  NB NB this should come from config
```

Comment:
Nice catch figuring out the bug with site resetting. I have a few minor formatting comments, mostly with spacing, but we can fix those with a linter later.